dev/io.openliberty.data.internal_fat_ddlgen/fat/src/test/jakarta/data/ddlgen/DDLGenTest.java

```diff
@@ -69,12 +68,19 @@ public static void setUp() throws Exception {
         DDLGenScriptResult result = DDLGenScript.build(server)
                         .execute()
                         .assertSuccessful()
+                        .assertDDLFile("application[DDLGenTestApp].module[DDLGenTestApp.war].databaseStore[java.comp.DefaultDataSource]_JakartaData.ddl")
+                        .assertDDLFile("application[DDLGenTestApp].databaseStore[TestDataSource]_JakartaData.ddl")
+                        .assertDDLFile("application[DDLGenTestApp].databaseStore[jdbc.TestDataSourceJndi]_JakartaData.ddl")
```

Comment:
Good question.  I'll add that test scenario in an additional pull request. My initial assumption is that they would go to different files, since we use the `dataStore` attribute of `@Repository` to create the `databaseStore` config.id at runtime.